xts/src/main/java/org/jboss/as/xts/XTSSubsystemTransformers.java

```diff
@@ -52,8 +52,10 @@ protected boolean isValueDiscardable(PathAddress address, String attributeName,
                    return attributeValue.isDefined() && attributeValue.equals(XTSSubsystemDefinition.HOST_NAME.getDefaultValue());
                }
            }, XTSSubsystemDefinition.HOST_NAME)
-           .setDiscard(new DiscardAttributeChecker.DiscardAttributeValueChecker(new ModelNode(false)), XTSSubsystemDefinition.DEFAULT_CONTEXT_PROPAGATION)
-           .addRejectCheck(RejectAttributeChecker.DEFINED, XTSSubsystemDefinition.HOST_NAME, XTSSubsystemDefinition.DEFAULT_CONTEXT_PROPAGATION)
+           .setDiscard(new DiscardAttributeChecker.DiscardAttributeValueChecker(new ModelNode(false)),
+                   XTSSubsystemDefinition.DEFAULT_CONTEXT_PROPAGATION, XTSSubsystemDefinition.ASYNC_REGISTRATION)
+           .addRejectCheck(RejectAttributeChecker.DEFINED, XTSSubsystemDefinition.HOST_NAME,
+                   XTSSubsystemDefinition.DEFAULT_CONTEXT_PROPAGATION, XTSSubsystemDefinition.ASYNC_REGISTRATION)
            .end();
 
            TransformationDescription.Tools.register(builder.build(), subsystem, ModelVersion.create(1, 1, 0));
```

Comment:
You also need to transform for the attribute for old version 2.1.0. I recommend looking for usage of ChainedTransformationDescriptionBuilder to make it easier to add transformers in one place going forward.